src/components/Partners/BusinessFieldFilter.tsx

```diff
@@ -72,29 +105,55 @@ const BusinessFieldFilter: React.FC<BusinessFieldFilterProps> = ({
                 },
             }}
         >
-            <MenuItem sx={{ display: 'none' }} value={'default'}>
-                <Typography variant="caption">Business fields</Typography>
-            </MenuItem>
-            {state.businessField.map((businessField, index) => (
-                <MenuItem key={index} value={businessField.name}>
-                    <ListItemText
-                        // sx={{ color: !businessField.active ? '#CBD4E2' : '#ffffff' }}
-                        primary={
-                            <Typography
-                                variant="caption"
-                                sx={{ fontWeight: !businessField.active ? 500 : 600 }}
-                            >
-                                {businessField.name}
-                            </Typography>
-                        }
-                    />
-                    {businessField.active && (
-                        <Icon path={mdiCheckCircle} size={1} color="#B5E3FD" />
-                    )}
-                </MenuItem>
-            ))}
+            {groupedBusinessFields.flatMap((group, groupIndex) => [
```

Comment:
The business fields should be grouped by the first element of each business field, not by the complete name